web/client/plugins/Search.jsx

```diff
@@ -99,14 +100,17 @@ const ToggleButton = require('./searchbar/ToggleButton');
  *        "sortBy": "ID",
  *        "srsName": "EPSG:4326",
  *        "maxFeatures": 4,
- *        "blackist": [... an array of strings to exclude from the final search filter ]
+ *        "blackist": [... an array of strings to exclude from the final search filter ],
+ *        "returnFullResponse": false
```

Comment:
`returnFullResponse` is not a property that should be configured in search plugin. 
This example is wrong and seems to suggest to have more info, instead it breaks the functionality at all. 
Document the property in the right place.